css/styles.css

```diff
@@ -0,0 +1,680 @@
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --black-color: #272a31;
+  --red-color: #ec5242;
+  --gray-color: #d3d3d3;
+  --nav-color: #f7f4f4;
+}
+
+a:hover {
+  cursor: pointer;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bar {
+  display: block;
+  width: 43px;
+  height: 8px;
+  margin: 3px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.nav-link .campain {
+  border: #94544d solid;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 15px;
+}
+
+.logo img {
+  width: 10em;
+}
+
+.topbar img {
+  margin: -7px 0 4px 1px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #272a31;
+  margin-left: 32px;
+}
```

Comment:
- [ ] Kindly make sure that your nav links color is the same as the provided on on the design. :wink: 

   - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203581756-623f011a-c7b2-4366-97a2-6f2e9e6c420e.png)

   - ### PROVIDED DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203581838-17a8cde9-b568-448e-ae33-f2b80cc88480.png)